### PR TITLE
Update documentation for CommandWithMeta

### PR DESCRIPTION
**Pull request by montchr**
The docs for `get()` were missing and there was a typo in `delete()`.

### CODE REVIEW

File: php/WP_CLI/CommandWithMeta.php [hunk 1]
```diff
@@ -77,6 +77,15 @@ public function list_( $args, $assoc_args ) {
...
+	 * <id>
+	 * : The ID of the object.
+	 *
+	 * <key>
+	 * : The name of the meta field to get.
+	 *
+	 * [--format=<format>]
+	 * : Accepted values: table, json. Default: table
+	 *
 	 * @synopsis <id> <key> [--format=<format>]
```
Comment (danielbachhuber) [inline php/WP_CLI/CommandWithMeta.php]: Can you remove `@synopsis` ? Now that we have expanded docs here, it's no longer necessary.

**Comment by danielbachhuber** [comment]
Thanks — one small note to address.

**Comment by danielbachhuber** [comment]
You'll want to merge `master` too, so the build doesn't fail.

**Comment by montchr** [comment]
Okay, all set.

**Comment by danielbachhuber** [comment]
:palm_tree:

**Pull request merged**